app/controllers/alerts_list_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 # Controller for showing Alert records
 class AlertsListController < ApplicationController
+  include AlertsHelper
```

Comment:
yes! u are right it says that!! But I don't know why it still works for me when I am not including the helper module in the controller. Would be great if someone has additional input on this.